common/utils.js

```diff
@@ -213,7 +213,8 @@ export async function setPolyfillBackend(device) {
   // Note: 'wasm' backend may run failed on some samples since
   // some ops aren't supported on 'wasm' backend at present
   const backend = device === 'cpu' ? 'wasm' : 'webgl';
-  const tf = navigator.ml.createContext().tf;
+  const ml = await navigator.ml.createContext();
```

Comment:
Here variable `ml` may not be descriptive. 
How about renaming it as `mlContext` or `context`?

Reply:
Good catch! Thanks!
Fixed in new commit.